# see for all field types:
# https://index.ros.org/doc/ros2/Concepts/About-ROS-Interfaces/
# <fieldtype> <fieldname>:
string first_name
string last_name
uint8 age
string address